doc/nrf/gs_installing.rst

```diff
@@ -355,34 +371,34 @@ Use the following commands to install the requirements for each repository.
 
 .. rst-class:: numbered-step
 
-Install the toolchain
-*********************
+Install the GNU Arm Embedded Toolchain
+**************************************
 
 To be able to cross-compile your applications for Arm targets, you must install version 9-2019-q4-major of the `GNU Arm Embedded Toolchain`_.
 
 .. important::
-   Make sure to install the version that is mentioned above.
-   Other versions might not work with this version of the |NCS|.
-
-   Other versions of the |NCS| might require a different toolchain version.
+   Make sure to install the toolchain version that is mentioned above.
+   Other toolchain versions might not work with this version of the |NCS|.
+   Similarly, other versions of the |NCS| might require a different toolchain version.
 
 To set up the toolchain, complete the following steps:
 
 .. _toolchain_setup:
 
 1. Download the `GNU Arm Embedded Toolchain`_ for your operating system.
-#. Extract the toolchain into a folder of your choice.
-   We recommend to use the folder ``c:\gnuarmemb`` on Windows and ``~/gnuarmemb`` on Linux or macOS.
-
+#. Extract the contents of the root folder of the toolchain into a directory of your choice.
+   The recommended folder is :file:`c:\gnuarmemb` on Windows and :file:`~/gnuarmemb` on Linux or macOS.
    Make sure that the folder name does not contain any spaces or special characters.
-#. If you want to build and program applications from the command line, define the environment variables for the GNU Arm Embedded toolchain.
+#. If you want to build and program applications from the command line, define the environment variables for the GNU Arm Embedded Toolchain.
    Depending on your operating system:
 
     .. tabs::
 
        .. group-tab:: Windows
 
-          Open a command prompt and enter the following commands (assuming that you have installed the toolchain to ``c:\gnuarmemb``; if not, change the value for GNUARMEMB_TOOLCHAIN_PATH):
+          Open a command-line window with administrator rights and enter the commands below.
```

Comment:
Do you need administrator rights here? And if you set the variables as administrator, do they apply when you build as user?

Reply:
I've researched the topic and discussed f2f with Ruth. I'll remove the admin rights mention, as it is not needed until proved otherwise.